doc/cprover-manual/contracts-assigns.md

```diff
@@ -59,17 +153,18 @@ int sum(const uint32_t a, const uint32_t b, uint32_t* out)
 __CPROVER_assigns(*out, err_signal)
 ```
 
-### Semantics
+## Semantics
```

Comment:
Now this is really useful. Perhaps this needs to be put near the beginning of the document? As is, a lot of syntax is discussed, but the reader has no idea what all this means. See also my note on function parameters above: the statement in the document makes a lot more sense once the semantics have been described.